data/mapcss-mapping.csv

```diff
@@ -1053,7 +1053,7 @@ historic|wayside_shrine;965;
 historic|boundary_stone;966;
 historic|citywalls;967;
 historic|tomb;968;
-natural|rock;969;
+natural|rock;[natural=rock],[natural=stone];;;;969;
```

Comment:
replace_tags.txt ?

Reply:
Its better to have it in the main mapping file, its more explicit.

I see the `replaced_tags.txt` as a workaround for cases when additional matching is too long to be added directly to `mapcss-mapping`